rust_src/src/vectors.rs

```diff
@@ -139,25 +139,28 @@ impl LispVectorRef {
     pub fn as_slice(&self) -> &[LispObject] {
         unsafe {
             slice::from_raw_parts(
-                mem::transmute::<_, *const LispObject>(&self.contents),
+                &self.contents as *const [i64; 1] as *const LispObject,
+                //mem::transmute::<_, *const LispObject>(&self.contents),
```

Comment:
Why is this commented out?

Reply:
I  need to remove it. I had left it as reference while testing.